auth0/management/users.py

```diff
@@ -27,37 +33,37 @@ class Users:
 
     def __init__(
         self,
-        domain,
-        token,
-        telemetry=True,
-        timeout=5.0,
-        protocol="https",
-        rest_options=None,
-    ):
+        domain: str,
+        token: str,
+        telemetry: bool = True,
+        timeout: TimeoutType = 5.0,
+        protocol: str = "https",
+        rest_options: RestClientOptions | None = None,
+    ) -> None:
         self.domain = domain
         self.protocol = protocol
         self.client = RestClient(
             jwt=token, telemetry=telemetry, timeout=timeout, options=rest_options
         )
 
-    def _url(self, id=None):
+    def _url(self, id: str | None = None) -> str:
         url = f"{self.protocol}://{self.domain}/api/v2/users"
         if id is not None:
             return f"{url}/{id}"
         return url
 
     def list(
         self,
-        page=0,
-        per_page=25,
-        sort=None,
-        connection=None,
-        q=None,
-        search_engine=None,
-        include_totals=True,
-        fields=None,
-        include_fields=True,
-    ):
+        page: int = 0,
+        per_page: int = 25,
+        sort: str | None = None,
+        connection: str | None = None,
+        q: str | None = None,
+        search_engine: str | None = None,
+        include_totals: bool = True,
+        fields: List[str] | None = None,
+        include_fields: bool = True,
+    ) -> List[dict[str, Any]]:
```

Comment:
I believe this is the wrong type? I think it should be `Dict[str, Any]`

Reply:
@lilidworkin thanks. See https://github.com/auth0/auth0-python/pull/497#discussion_r1282096676 for more details